File: Exercicios-Beecrowd/Kotlin/1020.kt
import java.util.*
import java.util.Scanner

fun main(args: Array<String>) {

    val scanner=Scanner(System.`in`)
    val dias=scanner.nextInt()
    val ano=dias/365
    val mes=(dias%365)/30
    val dia=(dias%365)%30
    println("$ano ano(s)")
    println("$mes mes(es)")
    println("$dia dia(s)")
	
}

File: Exercicios-Beecrowd/C#/1020.cs
using System; 

class URI {

    static void Main(string[] args) { 

            int tempo = Convert.ToInt32(Console.ReadLine());
            int ano = tempo / 365;
            int mes = (tempo % 365)/30;
            int dia = ((tempo % 365) % 30);
            Console.WriteLine("{0} ano(s)", ano);
            Console.WriteLine("{0} mes(es)", mes);
            Console.WriteLine("{0} dia(s)", dia);

    }

}